File: Problem 002/solution.py
for _ in range(int(input())):
    N = int(input())
    a, b = 1, 2
    total_sum = 0
    while a < N:
        if a % 2 == 0:
            total_sum += a
        a, b = b, a + b

    print(total_sum)

File: Problem 002/solution.go
package main

import "fmt"

func main() {
    var t int
    fmt.Scan(&t)
    for i := 0; i < t; i++ {
        var n int
        fmt.Scan(&n)
        a := 1
        b := 2

        total_sum := 0
        for a < n {
            if a % 2 == 0 {
                total_sum += a
            }
            a, b = b, a + b
        }
        fmt.Println(total_sum)
    }
}
